Fix #3411: Merge administrator_controls_download_permissions_view into single xml file.

The-Anton opened the pull request:
## Explanation
Fixes #3411: Merged three ``` administrator_controls_download_permissions_view.xml ``` file (earlier created for mobile & tablet screens) into a single file. Variables for all the three different XML files are created and stored in ``` dimen.xml ```.

@rt4914  PTAL

Screenshots:

### Mobile-portrait

Before - 

<img src="https://user-images.githubusercontent.com/51144829/125042952-11131680-e0b8-11eb-8920-79026f84f189.jpg" width="20%" height="20%">

After - 

<img src="https://user-images.githubusercontent.com/51144829/125043809-f3927c80-e0b8-11eb-9a65-a0140d60f046.jpg" width="20%" height="20%">

### Mobile-landscape

Before - 

<img src="https://user-images.githubusercontent.com/51144829/125045919-22a9ed80-e0bb-11eb-9f76-42c56ffa158a.jpg" width="50%" height="50%">

After - 

<img src="https://user-images.githubusercontent.com/51144829/125045981-30f80980-e0bb-11eb-9f38-8e29874fe1df.jpg" width="50%" height="50%">

### Tablet-portrait

Before - 

<img src="https://user-images.githubusercontent.com/51144829/125046330-89c7a200-e0bb-11eb-8560-5cd90c1cba5d.png" width="50%" height="50%">

After - 

<img src="https://user-images.githubusercontent.com/51144829/125046412-9d730880-e0bb-11eb-9eb6-7be3f9e61169.png" width="50%" height="50%">


### Tablet-landscape

Before - 

<img src="https://user-images.githubusercontent.com/51144829/125046527-ba0f4080-e0bb-11eb-83ec-d1f4737d2068.png" width="50%" height="50%">

After - 

<img src="https://user-images.githubusercontent.com/51144829/125384865-29847900-e3b7-11eb-97a9-3f0ab842c581.png" width="50%" height="50%">

## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

The-Anton committed:
Merge administrator_controls_download_permissions_view into single xml file

The-Anton committed:
Merge branch 'develop' into combine-admin-permission-multi-layout

# Conflicts:
#	app/src/main/res/values-land/dimens.xml
#	app/src/main/res/values-sw600dp-port/dimens.xml
#	app/src/main/res/values/dimens.xml

The-Anton commented:
@rt4914 PTAL

rt4914 commented:
@The-Anton Apologies for delay. I will review it within next 24 hours. Meanwhile can you see that the last screenshot that you have added does not look like before screenshot, so can you please check that and fix it.

rt4914 reviewed (approved):
LGTM, thanks,

rt4914 commented:
@The-Anton Can you just merge your PR with latest develop so that the CI checks can pass and we can merge it.

The-Anton committed:
Merge branch 'develop' into combine-admin-permission-multi-layout

The-Anton committed:
Fixed tablet landscape view padding for administator_control_download_permission_view.xml

The-Anton commented:
@rt4914 Thanks for reviewing it. I have updated the code for the tablet landscape screen so that there is no difference in the UI. The image in the PR description is also updated.

The-Anton committed:
Adding runConfigurations.xml, misc.xml and build.gradle file to commit

rt4914 commented on .idea/misc.xml:
```diff
@@ -1,9 +1,9 @@
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Please revert this change. It should not be part of this PR.

The-Anton replied:
Done.

rt4914 commented on .idea/runConfigurations.xml:
```diff
@@ -3,10 +3,11 @@
     </option>
   </component>
-</project>
+</project>
```
Please revert this change. It should not be part of this PR.

The-Anton replied:
Done

rt4914 commented on build.gradle:
```diff
@@ -8,7 +8,7 @@ buildscript {
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.2.2'
```
Please revert this change. It should not be part of this PR.

The-Anton replied:
Done

rt4914 reviewed (changes requested):
@The-Anton PTAL. There are some unintentional changes. Please revert them. Thanks.

The-Anton committed:
Reverting configuration files

The-Anton committed:
Merge branch 'develop' into combine-admin-permission-multi-layout

rt4914 commented:
> Hi, @The-Anton, the LGTM Label has been removed because the changes were requested on this PR. Thanks!.

@The-Anton Please do not resolve any conversation/comment threads. Instead you can reply on them and the person who has started those comment threads will resolve them. Thanks.

The-Anton commented:
Sorry, It was a mistake.
